fimsauthz-api/fims-authz.yaml

```diff
@@ -70,9 +72,26 @@ paths:
                     The requested scope(s), a list of space-delimited,
                     case-sensitive names. Same as in RFC 6749.
 
-                    Version 1.0.2 of this specificaiton allows a pair of scope for this case only:
+                    As of Version 1.0.2 of this specificaiton allows a pair of scope for this case only:
                     [dss.read.identification_service_areas, rid.read.enhanced_details]
                   type: string
+                resource:
+                  description: |
+                    The base URL of the server with which the resulting access token 
```

Comment:
I recommend making this the hostname of the server; I don't think we get any benefit from being more granular than that.  Also, perhaps a bit more precise: "The hostname of the server to which the resulting access token will be sent."

Reply:
The major reasoning here was that `url` is a format that is cleanly understood as an OpenAPI `format`.

We can use `format: hostname` or leave format off completely.

Reply:
Going with just the hostname does open up to a potential conflict if we have an org that is hosting multiple servers under the same hostname - i.e. the `auds` to both `example.com/dss` and `example.com/uss` shall be `example.com`. Would re-using the token to talk to both the services be reasonable?

Reply:
Thanks @arkits .  This may be a legit concern, if `example.com/uss` is a bad actor or compromised.  The attack would be `example.com/uss` receiving a token with `example.com` as the `aud` from another uss (say `victim-uss.co`), then `example.com/uss` could use that token for `DELETE` calls (or other empty-HTTP-body calls) to `example.com/dss` posing as `victim-uss.co`.

Of course `example.com/dss` if compromised could be deleting all kinds of stuff without getting API calls from anyone.  But if only `example.com/uss` is compromised and not `example.com/dss` then this seems like a possible attack.

@arkits does this capture your point?

I don't know that we have to solve this at the moment, but it should be captured for discussion and future resolution.